O/onnxruntime/build_tarballs.jl

```diff
@@ -0,0 +1,86 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "onnxruntime"
+version = v"1.10.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/microsoft/onnxruntime.git", "0d9030e79888d1d5828730b254fedc53c7b640c1")
+]
+
+# Bash recipe for building across all platforms
+script = """
+onnxruntime_version=$version
+""" * raw"""
+cd $WORKSPACE/srcdir
+
+if [[ $target == *-w64-mingw32 || $target == aarch64-apple-darwin ]]; then
+    if [[ $target == x86_64-w64-mingw32 ]]; then
+        onnxruntime_dist_name=onnxruntime-win-x64-$onnxruntime_version.zip
+    elif [[ $target == i686-w64-mingw32 ]]; then
+        onnxruntime_dist_name=onnxruntime-win-x86-$onnxruntime_version.zip
+    elif [[ $target == aarch64-apple-darwin ]]; then
+        onnxruntime_dist_nameonnxruntime-osx-arm64-$onnxruntime_version.tgz
+    fi
+
+    wget https://github.com/microsoft/onnxruntime/releases/download/v$onnxruntime_version/$onnxruntime_dist_name
```

Comment:
Don't download files during the build, they can't be cached. Add them as additional sources

Reply:
Fixed